std_msgs/String robot_name
